lib/src/common/cache/cache.dart

```diff
@@ -294,6 +301,66 @@ class Cache<TIdentifier, TValue> extends Object with Disposable {
     return new Future.value();
   }
 
+  /// Provides access to a value stored in the cache without informing the
+  /// [CachingStrategy] of a get.
+  ///
+  /// [weakGet] can be thought of sort of like a weak reference when working with
+  /// a garbage collector. [weakGet] won't prevent removal of a [TValue]
+  /// [TIdentifier] pair from the [Cache], [get] or [getAsync] will. In other
```

Comment:
> [weakGet] won't prevent removal of a [TValue] [TIdentifier] pair from the [Cache], [get] or [getAsync] will

Could you elaborate on how these prevent removal?

Reply:
Can do.